app/views/choose.nunjucks

```diff
@@ -3,6 +3,34 @@
 {% block header %}
     <h1 class="local-header--title--question">How do you want to get a test?</h1>
 {% endblock %}
-
 {% block content %}
+    {% if errorMessage %}
+    <div class="error-summary">
+        <h2 class="local-header--error error-summary-heading">
+            {{ errorMessage }}
+        </h2>
+    </div>
+    {% endif %}
+
+    <form method="get" class="form" action="{{ SITE_ROOT }}/location">
+        <div class="reading-width">
+            {% if errorMessage %}
+            <div class="form-group error">
```

Comment:
would be cleaner to write out the additional error class in a single `div` if there is an error, rather than repeat the whole `div`.